README.md

```diff
@@ -1 +1,46 @@
-# ToDo-Review
\ No newline at end of file
+# Today's To Do
```

Comment:
It is the best practice to add a clear title for your project which should be followed by a detailed description of it. This helps anyone who will visit your repo to get a clear understanding of your project. 😃